source/istio_virtualservice.go

```diff
@@ -36,6 +36,10 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
 
+	antns "sigs.k8s.io/external-dns/source/annotations"
+
+	"sigs.k8s.io/external-dns/source/utils"
+
```

Comment:
I would drop the whitespaces here and let gofmt/goimports group the imports

Reply:
done